internal/hcsoci/resources_wcow.go

```diff
@@ -41,9 +41,10 @@ func allocateWindowsResources(coi *createOptionsInternal, resources *Resources)
 	if coi.Spec.Root == nil {
 		coi.Spec.Root = &specs.Root{}
 	}
-	if coi.Spec.Root.Path == "" && (coi.HostingSystem != nil || coi.Spec.Windows.HyperV == nil) {
+
+	if coi.Spec.Root.Path == "" {
```

Comment:
I think the gcs would fail this case. Do we need to support not using a fully qualified path here?

Reply:
Ah, it gets filled in a few lines later after the mountContainerLayers call.